src/redux/books/books.js

```diff
@@ -3,27 +3,32 @@ const ADD = 'add';
 const DEL = 'del';
 
 // Empty array of books
-const initialState = [{
-  id: 1,
-  genre: 'genre',
-  title: 'book title',
-  author: 'author',
-  status: 'complete',
-  currentChapter: 'Current chapter',
-  chapterNumber: 'Chapter #number',
-},
+const initialState = [
+  {
+    id: 1,
+    genre: 'genre',
+    title: 'book title',
+    author: 'author',
+    status: 'complete',
+    currentChapter: 'Current chapter',
+    chapterNumber: 'Chapter #number',
+  },
 ];
 
 // Reducer
 export default function booksReducer(state = initialState, action = {}) {
   switch (action.type) {
     case ADD:
-      return [...state, {
-        ...action.payload,
-      }];
+      return [
+        ...state,
+        {
+          ...action.payload,
+        },
+      ];
     case DEL:
       return [...state.filter((book) => book.id !== action.payLoad.id)];
-    default: return state;
+    default:
+      return state;
```

Comment:
- Hello @asifaminisonline, you have not properly connected your app to the API. There is no `createAsyncThunk` used in the project for creating thunk action creators 🙏
- Please ensure that all the tasks in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md) have been fully achieved. Please fix this 🙏